packages/twenty-server/src/database/typeorm/metadata/migrations/1726757368824-migrationDebt.ts

```diff
@@ -0,0 +1,53 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class MigrationDebt1726757368824 implements MigrationInterface {
```

Comment:
This is not directly related to this PR but it reflects that migrations <> typeorm entities were out of sync. I ran this a few weeks ago when opening the PR but it seems the debt hasn't been picked up by anyone since

Reply:
Thanks! Ideally we should enforce a rule (danger?)
If *.entity.ts files are updated, a migration should be generated 🤔